spec/models/court_case_spec.rb

```diff
@@ -68,6 +68,41 @@
     end
   end
 
+  describe 'generated clean_case_number column' do
```

Comment:
you can reuse the same specs for all of these
https://rspec.info/features/3-12/rspec-core/example-groups/shared-examples/
the only difference should be the model you're creating and the name of the case number column. You can make thoes variables using lets.